Modelica/Math/BooleanVectors.mo

```diff
@@ -2,16 +2,12 @@ within Modelica.Math;
 package BooleanVectors "Library of functions operating on Boolean vectors"
  extends Modelica.Icons.Package;
 function allTrue
-    "Returns true, if all elements of the Boolean input vector are true ('and')"
+    "Returns true, if Boolean input vector is non-empty and all elements are true ('and')"
   extends Modelica.Icons.Function;
   input Boolean b[:] "Boolean vector";
-  output Boolean result "= true, if all elements of b are true";
+  output Boolean result = size(b, 1) > 0 and min(b) "= true, if all elements of b are true";
 algorithm
```

Comment:
Is empty algorithm section still needed?

Reply:
You are right; it shouldn't be:
```suggestion
```

Reply:
In all three places, pls.

Reply:
Yes, of course.  Done.